nodemapper/src/gui/Builder/components/Flow.tsx

```diff
@@ -140,81 +141,118 @@ const ModuleNode = (props: NodeProps<ModuleData>) => {
   }
 
   return (
-    <div
-      className={styles.Node}
-      style={{
-        backgroundColor: props.data.color,
-      }}
-    >
-      <div className={styles.HeaderPanel}>
-        <div className={styles.HeaderText}>{props.data.config.name}</div>
-      </div>
-      {!named_inputs ? (
-        <>
-          {input_namespaces.length == 1 ? (
-            <Handle
-              className={styles.HandleInput}
-              id={input_namespaces[0]}
-              key={input_namespaces[0]}
-              type="target"
-              position={Position.Left}
-              style={{ top: "50%" }}
-            />
-          ) : null}
-          <Handle
-            className={styles.HandleOutput}
-            id="out"
-            type="source"
-            position={Position.Right}
-            style={{ top: "50%" }}
-          />
-        </>
-      ) : (
-        <div
-          className={styles.BodyPanel}
-          style={{
-            height: `${input_namespaces.length * 18 - 4}px`,
-          }}
-        >
-          {input_namespaces.map((name) => (
-            <div key={"div-" + name}>
+    <>
+      <div
+        className={styles.Node}
+        style={{
+          backgroundColor: props.data.color,
+        }}
+      >
+        {selected ? (
+          <NodeResizeControl
+            style={nodeResizeControlStyle}
+            minWidth={120}
+            minHeight={50}
+            variant={ResizeControlVariant.Line}
+            position="top-right"
+            shouldResize={(e, params) => params.direction[1] === 0}
+          >
+            <span
+              style={{
+                float: "right",
+                color: "#bebebe",
+              }}
+            >
+              <FontAwesomeIcon icon={faLeftRight} />
+            </span>
+          </NodeResizeControl>
+        ) : null}
+        <div className={styles.HeaderPanel}>
+          <div className={styles.HeaderText}>{props.data.config.name}</div>
+        </div>
+        {!named_inputs ? (
+          <>
+            {input_namespaces.length == 1 ? (
```

Comment:
```suggestion
            {input_namespaces.length == 1 && (
```
and drop ` : null` after the block